app/src/main/java/com/framgia/fsalon/screen/scheduler/detail/BookingDetailViewModel.java

```diff
@@ -322,6 +334,13 @@ public void updateImage(String pathOrigin) {
         }
     }
 
+    @Override
+    public void onAddPhoto(LinearLayout layoutPhoto, View view) {
+        FloatingActionMenu fapMenu = (FloatingActionMenu) view.getParent();
+        fapMenu.close(true);
+        layoutPhoto.setVisibility(View.VISIBLE);
```

Comment:
WARNING: [UnnecessaryFullyQualifiedName] Unnecessary use of fully qualified name 'View.VISIBLE' due to existing static import 'android.view.View.VISIBLE'
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/imports.html#UnnecessaryFullyQualifiedName